auth/src/main/java/org/aerogear/mobile/security/SyncSecurityCheckExecutor.java

```diff
@@ -21,7 +21,7 @@
 public class SyncSecurityCheckExecutor extends AbstractSecurityCheckExecutor<SyncSecurityCheckExecutor> {
 
     public static class Builder extends SecurityCheckExecutor.Builder.AbstractBuilder<Builder, SyncSecurityCheckExecutor> {
-        Builder(final Context ctx) {
+        public Builder(final Context ctx) {
```

Comment:
I did this because this useful chaining interface wasn't exposed.

Currently https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/src/main/java/org/aerogear/mobile/security/SecurityService.java#L45 allows supplying only the `Context` and there's no way to give it the metrics service.
We could improve the `SecurityService` interface to allow supplying the other builder params instead, but that defeats the purpose of having a builder.

Reply:
It is meant to be retrieved from the SecurityCheckExecutor class through the `newSyncExecutor` and `newAsyncExecutor` methods.

Reply:
An Example usage can be found here https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/src/test/java/org/aerogear/mobile/security/SecurityCheckExecutorTest.java#L53